files/jcl/ZWESECUR.jcl

```diff
@@ -1,442 +1,443 @@
+//*    Zowe started task name.                                                  
+//*                                                                             
+//* 9) Update the SET ZSSSTC= statement to match the desired                    
+//*    ZSS started task name.                                                   
+//*                                                                             
+//* 10) Update the SET AUXSTC= statement to match the desired                   
+//*    ZSS Auxilary started task name.                                          
+//*                                                                             
+//* 11) Customize the commands in the DD statement that matches your            
+//*    security product so that they meet your system requirements.             
+//*                                                                             
+//* Note(s):                                                                    
+//*                                                                             
+//* 1. THE USER ID THAT RUNS THIS JOB MUST HAVE SUFFICIENT AUTHORITY            
+//*    TO ALTER SECURITY DEFINITONS                                             
+//*                                                                             
+//* 2. The Zowe started task user ID (variable ZOWEUSER) must be able           
+//*    to write persistent data in the zlux-app-server/deploy directory         
+//*    structure. This sample JCL makes the Zowe started task part of           
+//*    the ZOwe administrator group (SET STCGROUP=&ADMINGRP. statement)         
+//*    to achieve this goal. Another solution, which is provided                
+//*    commented out, is giving the Zowe started task CONTROL access to         
+//*    the UNIXPRIV SUPERUSER.FILESYS profile.                                  
+//*                                                                             
+//* 3. This job WILL complete with return code 0.                               
+//*    The results of each command must be verified after completion.           
+//*                                                                             
+//*********************************************************************         
+//         EXPORT SYMLIST=*                                                     
+//*                                                                             
+//         SET PRODUCT=RACF          * RACF, ACF2, or TSS                       
+//*                     12345678                                                
+//         SET ADMINGRP=ZWEADMIN     * group for Zowe administrators            
+//         SET STCGROUP=&ADMINGRP.   * group for Zowe started tasks             
+//         SET ZOWEUSER=ZWESVUSR     * userid for Zowe started task             
+//         SET ZSSUSER=ZWESIUSR      * userid for xmem started task             
```

Comment:
This should not be changed to ZSSUSER - ZSS is a component that runs under ZWESVUSR. Please revert this

Reply:
Agreed - the zss server is a C server that is part of the main ZWESVSTC started task.  It calls the cross memory server ZWESISTC, but there is no concept of a zss user.  There is the concept of a cross memory server user so XMEMUSER is better to use as the variable name in the JCL as this is what I've also called it in documentation

Reply:
I've made those changes now.